server/service/devices.go

```diff
@@ -259,29 +259,6 @@ func (svc *Service) FailingPoliciesCount(ctx context.Context, host *fleet.Host)
 	return 0, fleet.ErrMissingLicense
 }
 
-////////////////////////////////////////////////////////////////////////////////
-// Device API features
-////////////////////////////////////////////////////////////////////////////////
-
-type deviceAPIFeaturesRequest struct {
```

Comment:
Should we keep this one as the "Ping" request? (and return capabilities in headers like the others)

Reply:
so if we keep this one, and using Juan's example, we would first ping this endpoint, and use the capabilities we get, so we get rid of the temptation of using the `404` response directly.

I'm in if it makes sense for you folks

Reply:
+1, to not rely on 404s.

Reply:
cc: @juan-fdz-hawa just FYI, this is a slightly change in regards to the steps you outlined, I will be pushing a commit soon but wanted to let you know.

Reply:
@lucasmrod two details:

1. important: I will make this an unauthenticated endpoint, no need to provide a token
2. minor: sounds like it would be nice for this to be a `HEAD` request.